packages/react-core/src/components/DescriptionList/__tests__/DescriptionListHelpText.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { DescriptionListTermHelpText } from '../DescriptionListTermHelpText';
+
+import styles from '@patternfly/react-styles/css/components/DescriptionList/description-list';
+
+test('Renders to match snapshot', () => {
+  const { asFragment } = render(<DescriptionListTermHelpText>test</DescriptionListTermHelpText>);
+  expect(asFragment()).toMatchSnapshot();
+});
+
+test(`Renders default class ${styles.descriptionListTerm}`, () => {
+  render(<DescriptionListTermHelpText>test</DescriptionListTermHelpText>);
+  expect(screen.getByText('test')).toHaveClass(styles.descriptionListTerm);
```

Comment:
```suggestion
  expect(screen.getByText('test')).toHaveClass(styles.descriptionListTerm, {exact: true});
```

Also a test to check props spread correctly. This file name should also be updated to `DescriptionListTermHelpText`